open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,512 @@
+            start_time: startTime.format("HH:mm"),
+            end_time: endTime.format("HH:mm")
+        };
+
+        var dayIndex = _.indexOf(days, dayString);
+
+        if (_.isArray(sessionsStore[dayIndex])) {
+            sessionsStore[dayIndex].push(sessionObject);
+        } else {
+            sessionsStore[dayIndex] = [sessionObject]
+        }
+
+    });
+
+    _.each(tracks, function (track) {
+        var tracksObject = {
+            name: track.name,
+            id: track.id,
+            count: track.sessions.length
+        };
+        if (!_.contains(tracksStore, tracksObject)) {
+            tracksStore.push(tracksObject);
+        }
+    });
+
+
+    loadDateButtons();
+    callback();
+}
+
+function loadDateButtons() {
+    var dayButtonTemplate = $("#date-change-button-template").html();
+    var $dayButtonsHolder = $(".date-change-btn-holder");
+    var sortedDays = days.sort();
+    _.each(sortedDays, function (day, index) {
+        var $dayButton = $(dayButtonTemplate);
+        if (index == 0) {
+            $dayButton.addClass("active");
+        }
+        $dayButton.text(day);
+        $dayButtonsHolder.append($dayButton)
+    });
+    loadTracksToTimeline(sortedDays[0]);
+}
+
+function loadTracksToTimeline(day) {
+
+    var dayIndex = _.indexOf(days, day);
+    var trackTemplate = $("#track-template").html();
+    var sessionTemplate = $("#session-template").html();
+
+    var $tracksHolder = $(".track-container");
+    var $unscheduledSessionsHolder = $(".sessions-list");
+
+    $tracksHolder.html("");
+    $unscheduledSessionsHolder.html("");
+
+    var $trackElement = $(trackTemplate);
+    $trackElement.data("track-id", 0);
+    $trackElement.find(".track-header").html("Standalone &nbsp;&nbsp;&nbsp;<span class='badge'>0</span>");
+    $trackElement.find(".track-inner").css("height", time.unit.count * time.unit.pixels + "px");
+    $tracksHolder.append($trackElement);
+
+    _.each(tracksStore, function (track) {
+        var $trackElement = $(trackTemplate);
+        $trackElement.attr("data-track-id", track.id);
+        $trackElement.find(".track-header").html(track.name + "&nbsp;&nbsp;&nbsp;<span class='badge'>0</span>");
+        $trackElement.find(".track-inner").css("height", time.unit.count * time.unit.pixels + "px");
+        $tracksHolder.append($trackElement);
+    });
+
+    _.each(sessionsStore[dayIndex], function (session) {
+        var $sessionElement = $(sessionTemplate);
+        $sessionElement.data("session-id", session.id);
+        $sessionElement.attr("data-original-text", session.title);
+
+        var scheduled = false;
+        if (!_.isNull(session.start_time) && !_.isNull(session.end_time) && session.start_time != session.end_time) {
+            scheduled = true;
+        }
+
+        if (scheduled) {
+            $sessionElement.addClass("scheduled");
+            $sessionElement.attr("data-original-title", session.start_time + " to " + session.end_time);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'start_time' is not in camel case.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650806054/issues/source?bid=3233864&fileBranchId=3324174#l423)